Quaver.Shared/Screens/Gameplay/GameplayScreen.cs

```diff
@@ -863,7 +863,10 @@ private void PlayComboBreakSound()
                 return;
             }
 
-            if (LastRecordedCombo >= 20 && Ruleset.ScoreProcessor.Combo == 0)
+            var timeSincePlay = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds() - TimePlayed;
+            var lastRecCom = (timeSincePlay < 10000) ? 1 : 20;
```

Comment:
Handling this with absolute time isn't the best way to do it (if the user decides not to skip the intro, it won't work). 

Instead, I would just check if the player has 1 miss while there are less than 20 total judgements. This will fulfill the feature request properly.

Something along the lines of this:

```cs
if (processor.Combo == 0)
{
    var missedIntro = processor.TotalJudgementCount <= 20 && processor.CurrentJudgements[Judgement.Miss] == 1;

    if (LastRecordedCombo >= 20 || missedIntro)
        SkinManager.Skin.SoundComboBreak.CreateChannel().Play();
}
```